### ISSUE TITLE
Applying experimental flag streaming causes a internal server error on deployment. 

### ISSUE THREAD

**Issue by gtokman** [sst/open-next#308]
Passing in the experimental flag to support text streaming with an `route.ts` lambda results in the following crash in the screenshot.
```
 experimental: {
     streaming: true,
  }
```

<img width="1436" alt="Screenshot_2023-10-24_at_9 58 37_AM" src="https://github.com/sst/open-next/assets/12258850/3b3c000a-afd0-44f3-bcba-40711baee11a">

**Comment by khuezy** [comment]
Can you look at your CW logs for any useful errors?
What next version?

**Comment by gtokman** [comment]
@khuezy thanks for the suggestion.

I'm on 
```
  "sst": "^2.32.0",
   "constructs": "10.2.69",
   "next": "^13.5.3" 
```

**Comment by khuezy** [comment]
Check your Lambda Server Function's config and make sure the invoke type is set to STREAMING. If you're using `buildCommand`, make sure you're properly setting the `--streaming` flaga

**Comment by gtokman** [comment]
Okay, are you saying I should use `buildCommand` and stop using `npx sst deploy --stage prod`?

**Comment by khuezy** [comment]
No, those two commands are unrelated. Can you put up a min repro?

**Comment by gtokman** [comment]
@khuezy this is the config I'm using that causes the issue.

```
import {
  Certificate,
  CertificateValidation,
} from "aws-cdk-lib/aws-certificatemanager";
import { HostedZone } from "aws-cdk-lib/aws-route53";
import { SSTConfig } from "sst";
import { NextjsSite } from "sst/constructs";

export default {
  config(_input) {
    return {
      name: "web",
      region: "us-east-1",
      bootstrap: {
        stackName: "test-bootstrap-stack",
      },
    };
  },
  stacks(app) {
    app.stack(function Site({ stack }) {
      const hostedZone = HostedZone.fromLookup(stack, "HostedZone", {
        domainName: "test.com",
      });

      const certificate = new Certificate(stack, "Certificate", {
        domainName:
          stack.stage === "prod" ? "test.com" : `${stack.stage}.test.com`,
        subjectAlternativeNames:
          stack.stage === "prod" ? ["www.test.com"] : undefined,
        validation: CertificateValidation.fromDns(hostedZone),
      });

      const site = new NextjsSite(stack, "site", {
        edge: true,
        customDomain: {
          domainName:
            stack.stage === "prod" ? "test.com" : `${stack.stage}.test.com`,
          domainAlias: stack.stage === "prod" ? "www.test.com" : undefined,
          hostedZone: "test.com",
          cdk: {
            certificate,
            hostedZone,
          },
        },
        environment: {
          ... keys
        },
        experimental: {
          streaming: true,
        },
      });
      stack.addOutputs({
        SiteUrl: site.url,
      });
    });
  },
} satisfies SSTConfig;

```

**Comment by khuezy** [comment]
@gtokman edge lamdas do not support streaming unfortunately. I don't know if they'll support them in the future but I hear they have some updates on lambda soon...

**Comment by gtokman** [comment]
@khuezy got it, that makes sense. If we don't use edge, our website takes forever to load. So what would you recommend?

**Comment by conico974** [comment]
TLDR; lambda@edge does not support streaming
